static/app/components/modals/inviteMembersModal/inviteMembersFooter.tsx

```diff
@@ -0,0 +1,80 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ButtonBar from 'sentry/components/buttonBar';
+import InviteButton from 'sentry/components/modals/inviteMembersModal/inviteButton';
+import {useInviteMembersContext} from 'sentry/components/modals/inviteMembersModal/inviteMembersContext';
+import InviteStatusMessage from 'sentry/components/modals/inviteMembersModal/inviteStatusMessage';
+import {space} from 'sentry/styles/space';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface Props {
+  canSend: boolean;
+}
+
+export default function InviteMembersFooter({canSend}: Props) {
+  const organization = useOrganization();
+  const {
+    complete,
+    inviteStatus,
+    setInviteStatus,
+    invites,
+    pendingInvites,
+    sendInvites,
+    sendingInvites,
+    willInvite,
+  } = useInviteMembersContext();
+  const isValidInvites = invites.length > 0;
+
+  const removeSentInvites = () => {
+    const emails = Object.keys(inviteStatus);
+    let newInviteStatus = {};
+    emails.forEach(email => {
+      if (pendingInvites.emails.has(email)) {
+        newInviteStatus = {...newInviteStatus, [email]: inviteStatus[email]};
+      }
+    });
+    setInviteStatus(newInviteStatus);
+  };
+
+  return (
+    <FooterContent>
+      <div>
+        {/* TODO(mia): remove these props and use InviteMemberContext once old modal is removed */}
+        <InviteStatusMessage
+          data-test-id="invite-status-message"
+          complete={complete}
+          hasDuplicateEmails={false}
+          inviteStatus={inviteStatus}
+          sendingInvites={sendingInvites}
+          willInvite={willInvite}
+        />
+      </div>
+      <ButtonBar gap={1}>
+        <Fragment>
+          <InviteButton
+            invites={invites}
+            willInvite={willInvite}
+            size="sm"
+            data-test-id="send-invites"
```

Comment:
You want to use `data-test-id` as a last resort. Here your test selector should be `getByRole('button', {name: 'Send (50) Invites'})` (or whatever the label is).

The reason you want to write tests like this is that it's testing what the user **is actually seeing and doing**. vs when you have data test ids, the user doesn't click buttons based on those. So the label could totally break and the user wouldn't be able to use this anymore, but the test would still pass.

More reading about this: https://testing-library.com/docs/dom-testing-library/api-accessibility/

Reply:
Here I am testing to make sure that the correct status message is shown, so this text doesn't have a role. Would `getByText` the best way to check this, since there is no role?